jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -48,7 +48,7 @@ do
 done
 
 mkdir -p "${LOGS_DIR}/qemu"
-sudo sh -c "cp -r /var/log/libvirt/qemu/* "${LOGS_DIR}/qemu/""
+sudo sh -c "cp -r /var/log/libvirt/qemu/* \"${LOGS_DIR}/qemu/\""
```

Comment:
you could also remove the inner quotes. they were mostly for the variable, and that's now taken care by the outer quotes.

Reply:
done